1-your-first-cluster.md

```diff
@@ -393,6 +395,7 @@ Cluster API's CLI - `clusterctl` is used to install the CAPI Management Cluster
 
 ```bash
 $env:CLUSTERCTL_REPOSITORY_PATH = ([System.Uri](Get-Item .).FullName).AbsoluteUri + "/clusterctl/repository"
+$env:path = (Get-Item .).FullName + ';' + $env:path
```

Comment:
What does this set the path to?

Reply:
Just if someone else reads it, as discussed adds the absolute current path and it adds it to the beginning which gives our binaries higher prio

Previous variant only added `.` which kind of changes when moving around directories